connect-streams-pipeline/src/main/java/io/confluent/examples/connectandstreams/consoleproducer/StreamsIngest.java

```diff
@@ -18,28 +18,26 @@
 
 import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.kafka.common.serialization.Serdes;
-import org.apache.kafka.common.utils.Bytes;
 import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.StreamsConfig;
 import org.apache.kafka.streams.kstream.Consumed;
 import org.apache.kafka.streams.kstream.Grouped;
 import org.apache.kafka.streams.kstream.KStream;
-import org.apache.kafka.streams.kstream.Materialized;
 import org.apache.kafka.streams.kstream.Printed;
-import org.apache.kafka.streams.state.KeyValueStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.Properties;
 
 import io.confluent.examples.connectandstreams.avro.Location;
 
 public class StreamsIngest {
 
-  static final String INPUT_TOPIC = "consoleproducer-locations";
-  static final String DEFAULT_BOOTSTRAP_SERVERS = "localhost:9092";
-  static final String KEYS_STORE = "consoleproducer-count-keys";
-  static final String SALES_STORE = "consoleproducer-aggregate-sales";
```

Comment:
Same here

Reply:
also not used, I noticed them grayed out in my IDE